CHANGELOG.md

```diff
@@ -9,12 +9,16 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Added
 
-- Build of packages for Fedora 34 platform.
-
 ### Changed
 
 ### Fixed
 
+## [1.1.2] - 2022-08-21
+
+### Added
+
+- Build of packages for Fedora 34 platform. (Issue: #128 PR: #129).
```

Comment:
Let's add links only to issues, or in if there are no issues to PR. Usually from one you can find the other.
```suggestion
- Build of packages for Fedora 34 platform (#128).
```

Reply:
Updated.